Source/Utils/Make.package

```diff
@@ -12,6 +12,7 @@ CEXE_sources += WarpXAlgorithmSelection.cpp
 CEXE_headers += NCIGodfreyTables.H
 CEXE_headers += WarpX_Complex.H
 CEXE_headers += IonizationEnergiesTable.H
+CEXE_headers += WarpXProfilerWrapper.H
```

Comment:
I am afraid it would be best to include this header in all files where you want to use its macros :)

Reply:
That should be done now.